domain/schema/model/sql/0008-space.sql

```diff
@@ -19,3 +19,30 @@ ON provider_space (space_uuid);
 
 INSERT INTO space VALUES
 (0, 'alpha');
+
+CREATE VIEW v_space AS
```

Comment:
`v_space` and `v_subnet` are essentially the same. They should be a single view.

Reply:
I did this originally then backed it out because the order of the joins was different. Looking closer now, as far as I can tell it should be ok if we join subnets to space, since every space has a subnet.

I have used what was the space view as the view for both, since this is not going to return NULLs in unexpected places.